server/functions/__tests__/mocks/client/controllers.ts

```diff
@@ -19,4 +19,7 @@ export class ClientAuthController extends AuthControllerBase {
         throw new Error('Method not implemented.');
     }
 
-}
\ No newline at end of file
+    protected async servicesSignOut() {
+        return;
+    }
```

Comment:
Does this need to be implemented before merging?

Reply:
It actually needs to be there in order for the controller to satisfy the interface it inherits from. It is intentionally left as a stub given that this controller serves as a mock.